tests/CommonTestUtils/Epilogue.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.ComponentModel;
+
+namespace CommonTestUtils
+{
+    /// <summary>
+    /// This class supports TestCleanUpAttribute. See TestCleanUpAttribute.cs
+    /// for technical details.
+    /// </summary>
+    public static class Epilogue
+    {
+        private static object Sync = new();
+        private static OrderedDictionary<int, Action> AfterSuiteActions = new();
+        private static List<Action> AfterTestActions = new();
```

Comment:
<img width="514" height="74" alt="Image" src="https://github.com/user-attachments/assets/f0f1710a-197d-40d6-9e82-0281030d08b0" />

```suggestion
        private static object _sync = new();
        private static OrderedDictionary<int, Action> _afterSuiteActions = new();
        private static List<Action> _afterTestActions = new();
```